src/topology/metric_space/isometry.lean

```diff
@@ -123,6 +113,26 @@ lemma isometry.comp_continuous_iff {γ} [topological_space γ] (hf : isometry f)
   continuous (f ∘ g) ↔ continuous g :=
 hf.uniform_inducing.inducing.continuous_iff.symm
 
+end pseudo_emetric_isometry --section
+
+section emetric_isometry
+variables [emetric_space α]
+
+/-- An isometry from a metric space is a uniform embedding -/
+theorem isometry.uniform_embedding [pseudo_emetric_space β] {f : α → β} (hf : isometry f) :
+  uniform_embedding f :=
+hf.antilipschitz.uniform_embedding hf.lipschitz.uniform_continuous
+
+/-- An isometry from a complete emetric space is a closed embedding -/
+theorem isometry.closed_embedding [complete_space α] [emetric_space β]
+  {f : α → β} (hf : isometry f) : closed_embedding f :=
+hf.antilipschitz.closed_embedding hf.lipschitz.uniform_continuous
+
+lemma isometry.tendsto_nhds_iff [complete_space α] [emetric_space β] {ι : Type*} {f : α → β}
+  {g : ι → α} {a : filter ι} {b : α} (hf : isometry f) :
+  a.tendsto g (𝓝 b) ↔ a.tendsto (f ∘ g) (𝓝 (f b)) :=
```

Comment:
We typically don't use dot notation for `tendsto`. would you agree to
```suggestion
  tendsto a g (𝓝 b) ↔ tendsto a (f ∘ g) (𝓝 (f b)) :=
```
?

Reply:
Sure. It's just that `filter` isn't opened, so it would rather be
```suggestion
  filter.tendsto g a (𝓝 b) ↔ filter.tendsto (f ∘ g) a (𝓝 (f b)) :=
```